overlord/snapstate/snapstate_test.go

```diff
@@ -555,16 +555,16 @@ func (s *snapmgrTestSuite) TestInstallDevModeConfinementFiltering(c *C) {
 
 	// if a snap is devmode, you can't install it without --devmode
 	opts := &snapstate.RevisionOptions{Channel: "channel-for-devmode"}
-	_, err := snapstate.Install(s.state, "some-snap", opts, s.user.ID, snapstate.Flags{})
+	_, err := snapstate.Install(context.TODO(), s.state, "some-snap", opts, s.user.ID, snapstate.Flags{})
```

Comment:
unless the plan is to have some sort of context you can pass to these from the test, none of these are TODOs, and having so many of them make finding the ones we actually want a lot harder.